### PR TITLE
User login (Student/Teacher API): Returns JWT key when ever user log in

### PR DESCRIPTION
**Pull request by vubuing**

### CODE REVIEW

File: middlewares/auth.go [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+	if ts, err = t.SignedString([]byte(os.Getenv("AUTH_SIGNING_KEY"))); err != nil {
+		return "", err
+	}
+	return ts, nil
+}
+
+func IsAuth(c *gin.Context) {
+	auth := c.Request.Header.Get("Authorization")
+	ts := strings.Split(auth, " ")[1]
+	if auth == "" || ts == "" {
```
Comment (l4rzy) [inline middlewares/auth.go:44]: after assuring Authorization header is available, this checking is unnecessary, just process normally with `jwt.Parse`

**Review by l4rzy — CHANGES_REQUESTED** [review]
comments as below

File: middlewares/auth.go (hunk 1 above)
Comment (vubuing) [reply middlewares/auth.go:44]: image
I have tested this. If the Authorization header comes with "Bearer ", when we unchecked ts == "" condition, the token would be parsed failed.

Comment (l4rzy) [reply middlewares/auth.go]: m

**Pull request merged**